src/javascript/app/pages/user/account/portfolio/portfolio.init.js

```diff
@@ -32,8 +32,8 @@ const PortfolioInit = (() => {
         BinarySocket.send({ portfolio: 1 }).then((response) => {
             updatePortfolio(response);
         });
-        // Subscribe to transactions to auto update new purchases
-        BinarySocket.send({ transaction: 1, subscribe: 1 }, { callback: transactionResponseHandler });
+        // Wait for transactions to auto update on new purchases
+        BinarySocket.wait('transaction').then(transactionResponseHandler);
```

Comment:
The previous version (send) was passing every response to its callback, however by this change it passes only the last response once.